apps/frontend/src/services/users.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+import { UserWithRelatedData } from '@neighborhood/backend/src/types';
+import { StorageWithUser, UserInfo } from '../types';
+
+const baseURL = '/api/users';
+
+async function getUserData(id: number): Promise<UserWithRelatedData> {
```

Comment:
I think it's unnecessary to load the full user data, esp. the `user` and `responses` of each `Request`. We only need `neighborhoods`, and `requests` without the related data.

Reply:
We don't know. When I developed the home page, the request page wasn't finished yet, so the requests in the home page opened up the request modal, which needed additional request data.
Now that you've merged the Request branch, you are right, we don't need these anymore.

Reply:
Could we reduce the amount of data loaded now?

Reply:
Yes! I already did